src/cljs/org/broadinstitute/firecloud_ui/page/workspace/submission_details.cljs

```diff
@@ -126,9 +126,43 @@
                    (:workflows props))}])))})
 
 
+(react/defc AbortButton
+  {:kill-submission
+   (fn [{:keys [props state this]}]
```

Comment:
This isn't a case of that.  The `state` variable here is the component's own state.